README.md

```diff
@@ -1,3 +1,102 @@
-# rukpak
+# RukPak
 
-Rukpak runs in a Kubernetes cluster and defines an API for installing cloud native bundle content 
+RukPak runs in a Kubernetes cluster and defines an API for installing cloud native bundle content. 
```

Comment:
Any value in calling out the stage of this project (e.g. pre-alpha) and is subject to change?

Reply:
Let's handle this kind of work in a follow-up/further down the line.